### ISSUE TITLE
Uncaught ReferenceError: walletScoreListener is not defined

### ISSUE THREAD

**Issue by nicolaevladescu** [bitcoin-dot-org/bitcoin.org#1844]
[Bounty](https://github.com/bitcoin-dot-org/bitcoin.org#earn-bitcoin-for-contributing): **10,000 bits / ~$50 USD**

Hover the question mark icon at: https://bitcoin.org/en/wallets/desktop/windows/bitcoincore/

```
Uncaught ReferenceError: walletScoreListener is not defined
    at HTMLDivElement.onclick ((index):465)
onclick @ (index):465
```

`Google Chrome Version 61.0.3163.100 (Official Build) (64-bit)`
`Windows 10 Pro 1703`

P.S. Did a cross wide project search and there is no defined function named `walletScoreListener `

**Comment by Cobra-Bitcoin** [comment]
Previously a useful pop up would show when someone hovered over the question mark. @kuzzmi has worked a lot on the wallets stuff recently. He probably knows what's happening here.

Maybe we could put a bounty on a fix? @wbnns

**Comment by wbnns** [comment]
@Cobra-Bitcoin Sure, I just added a bounty.

@nicolaevladescu Nice catch and thanks for reporting!

**Comment by kuzzmi** [comment]
Should be fixed after merging #1855. My bad for missing this part.
